app/liquid_tags/medium_tag.rb

```diff
@@ -0,0 +1,46 @@
+class MediumTag < LiquidTagBase
+  include ApplicationHelper
+  include ActionView::Helpers::TagHelper
+  include InlineSvg::ActionView::Helpers
```

Comment:
Added this line for `inline_svg` to reuse the Medium icon that's already in codebase